### PR TITLE
Reworking to use new cryto lib

### PR DESCRIPTION
**Pull request by warrenbailey**
## What? and Why?
Use the updated sdc-cryptography lib

## Checklist
  - [x] CHANGELOG.md updated? (if required)

### CODE REVIEW

**Comment by warrenbailey** [comment]
Requires this change to be merged and release first https://github.com/ONSdigital/sdc-cryptography/pull/12

File: requirements.txt [hunk 1]
```diff
@@ -15,4 +15,4 @@ sdx-common==0.7.0
 six==1.10.0
 structlog==16.1.0
 Werkzeug==0.11.9
-sdc-cryptography==0.1.6
\ No newline at end of file
+sdc-cryptography==0.2.0-RC1
```
Comment (necrophonic) [inline requirements.txt]: Probably shouldn't be taking branches back to master that rely on release candidates - we certainly shouldn't release anything to prod that does.

Comment (warrenbailey) [inline requirements.txt]: I'm planning on setting it  to 0.2.0 when this change is merged: https://github.com/ONSdigital/sdc-cryptography/pull/12

File: Makefile [hunk 2]
```diff
@@ -1,7 +1,6 @@
 build:
-	git clone --branch 0.7.0 https://github.com/ONSdigital/sdx-common.git
-	pip3 install ./sdx-common
 	rm -rf sdx-common
+	cd .. && pip3 uninstall -y sdc-cryptography && pip3 install -I ./sdc-cryptography
```
Comment (warrenbailey) [inline Makefile]: Will remove once the sdc-crypto changes are merged

**Pull request merged**